assets/constants.ts

```diff
@@ -5493,6 +5493,10 @@ export default {
 
   "ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE": true,
 
+  // Maximum allowed length for generating IDs using get new ID method in
+  // in base models for storage models.
+  "BASE_MODELS_ID_LENGTH": 12,
```

Comment:
Why this need to be placed in constnats.ts? (I think in constnats.ts we only add constant which need to be used/shared in both frontend/backend and model_id_length doesn't look like the information needed for the frontend, right?)

Reply:
Removed.